packages/waggle/src/socket/IOProxy.ts

```diff
@@ -44,12 +44,7 @@ export default class IOProxy {
 
   public subscribeForTopic = async (peerId: string, channelData: IChannelInfo) => {
     log(`${peerId} is subscribing for channel ${channelData.address}`)
-    await this.getStorage(peerId).subscribeForChannel(channelData.address, channelData)
-  }
-
-  public updateChannels = async (peerId: string) => {
-    const channels = await this.getStorage(peerId).updateChannels()
-    this.io.emit(EventTypesResponse.RESPONSE_GET_PUBLIC_CHANNELS, channels)
+    await this.getStorage(peerId).subscribeForChannel(channelData)
```

Comment:
should be: subscribeToChannel

this feels really pedantic, and maybe i'm wrong in caring about this!

Reply:
That's ok. I'm aware of that. I just wasn't refactoring the names of existing methods

Reply:
it's probably good to do this if you see it and think it's easy.